test/distrib/python/distribtest.py

```diff
@@ -18,4 +18,4 @@
 # which is what we are testing here.
 channel = grpc.insecure_channel('localhost:1000')
 del channel
-print 'Success!'
+print('Success!')
```

Comment:
Debug line or additional signal?

Reply:
It turns out all our existing python distribtests use python2.7 (hence the change to make the distribtest work with python3)? I think we should fix that.